projects/packages/forms/src/contact-form/class-contact-form.php

```diff
@@ -365,19 +365,54 @@ public static function parse( $attributes, $content ) {
 			 * @param string $contact_form_id Contact form post URL.
 			 * @param $post $GLOBALS['post'] Post global variable.
 			 * @param int $id Contact Form ID.
+			 * @param int $page Page number.
 			 */
 			$url                     = apply_filters( 'grunion_contact_form_form_action', "{$url}#contact-form-{$id}", $GLOBALS['post'], $id, $page );
 			$has_submit_button_block = str_contains( $content, 'wp-block-jetpack-button' );
 			$form_classes            = 'contact-form commentsblock';
 			$post_title              = $post->post_title ?? '';
 			$form_accessible_name    = ! empty( $attributes['formTitle'] ) ? $attributes['formTitle'] : $post_title;
-			$form_aria_label         = isset( $form_accessible_name ) && ! empty( $form_accessible_name ) ? 'aria-label="' . esc_attr( $form_accessible_name ) . '"' : '';
+			$form_aria_label         = isset( $form_accessible_name ) && ! empty( $form_accessible_name ) ? esc_attr( $form_accessible_name ) : '';
 
 			if ( $has_submit_button_block ) {
 				$form_classes .= ' wp-block-jetpack-contact-form';
 			}
 
-			$r .= "<form action='" . esc_url( $url ) . "' method='post' class='" . esc_attr( $form_classes ) . "' $form_aria_label enctype='multipart/form-data' novalidate>\n";
```

Comment:
The one attribute we remove here that I don't see added via the new filter is 'novalidate'. To confirm, we don't want to keep that?

Reply:
Sorry, I'm wrong. It is there. So disregard this.